.bin/deploy-test-data.sh

```diff
@@ -124,14 +124,14 @@ _psql <<EOF
 begin;
 with rows as (
 insert into ggircs_portal.form_json
-  (id, name, form_json, prepopulate_from_swrs, prepopulate_from_ciip, form_result_init_function)
+  (id, name, slug, short_name, description, form_json, prepopulate_from_swrs, prepopulate_from_ciip, form_result_init_function)
   overriding system value
 values
-  (1, 'Admin', '$(cat "./data/portal/form_json/administration.json")'::jsonb, true, true, 'init_application_administration_form_result'),
-  (2, 'Emission', '$(cat "./data/portal/form_json/emission.json")'::jsonb, true, false, 'init_application_emission_form_result'),
-  (3, 'Fuel', '$(cat "./data/portal/form_json/fuel.json")'::jsonb, true, false, 'init_application_fuel_form_result'),
-  (4, 'Electricity and Heat', '$(cat "./data/portal/form_json/electricity_and_heat.json")'::jsonb, false, false, null),
-  (5, 'Production', '$(cat "./data/portal/form_json/production.json")'::jsonb, false, false, null)
+  (1, 'Administration Data', 'admin', 'Admin data', 'Admin description', '$(cat "./data/portal/form_json/administration.json")'::jsonb, true, true, 'init_application_administration_form_result'),
+  (2, 'Emission', 'emission', 'Emission', 'Emission description', '$(cat "./data/portal/form_json/emission.json")'::jsonb, true, false, 'init_application_emission_form_result'),
+  (3, 'Fuel','fuel', 'Fuel', 'Fuel description',  '$(cat "./data/portal/form_json/fuel.json")'::jsonb, true, false, 'init_application_fuel_form_result'),
+  (4, 'Electricity and Heat', 'electricity-and-heat', 'Electricity and Heat', 'Electricity and Heat description', '$(cat "./data/portal/form_json/electricity_and_heat.json")'::jsonb, false, false, null),
+  (5, 'Production', 'production', 'Production', 'Production description',  '$(cat "./data/portal/form_json/production.json")'::jsonb, false, false, null)
 on conflict(id) do update
 set name=excluded.name, form_json=excluded.form_json,
```

Comment:
I don't follow.

Reply:
The `on conflict(id) do update set` clause defines what should be done if we are trying to insert a row where the ID already exists. Unfortunately, there is no way to tell the database to just update the entire row, you have to explicitly tell it to update each field. Because you added `slug`, `short_name` and `description` to the table, you need to add them down here too (`slug=excluded.slug`, etc). `excluded` being the row that you are tring to insert.